src/components/MultiSelectCombobox/ComboboxSelections.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { Children, FC, MouseEventHandler } from 'react';
+import Button from '../Button/Button';
+import DropdownToggle from '../Dropdown/DropdownToggle';
+import Icon from '../Icon/Icon';
+import InputGroup from '../InputGroup/InputGroup';
+
+interface ComboboxSelectionsProps {
+  onRemoveAll?: () => void;
+  placeholder?: string;
+}
+
+const defaultProps = { placeholder: 'Click to select an option...' };
+
+const ComboboxSelections: FC<ComboboxSelectionsProps> = ({
+  children,
+  onRemoveAll,
```

Comment:
Nit: make this `lodash.noop` as well for consistency

Reply:
Instead of adding `noop` to props, I think it's a better pattern to just call these event functions like:
```javascript
onRemoveAll?.();
```
(which is just a nice way of doing the old, busted `onRemoveAll && onRemoveAll()`)

Any issues with that? Seems to be the cleanest.

Reply:
Yep, no strong opinion on this, either way is fine, just thought since other components in this PR use `noop`, might as well make this the same or vice versa. Another option (also my personal favorite) is `onRemoveAll = () => undefined`, basically `noop` without the import since `undefined` fits under `void`